src-tauri/Cargo.toml

```diff
@@ -19,6 +19,7 @@ serde_json = "1.0"
 reqwest = { version = "0.11.22", features = ["json"] }
 ts-rs = { version = "7.0", features = ["format"] }
 anyhow = "1.0.75"
+custom_error = "1.9.2"
```

Comment:
Might be better to use [thiserror](https://crates.io/crates/thiserror) since it is used more/better supported. Anyhow's (and [eyre](https://crates.io/crates/eyre) if we want something more high level a bit later) error types also work with it.

Reply:
Seems more popular indeed. I'll look into it.